test/case-long-stack/two.ts

```diff
@@ -0,0 +1 @@
+import * as two from "./two";
```

Comment:
What are you trying to test with these files?

Reply:
These validate that the `search-depth-limit` option is respected.

`one` imports from `two` imports from `three` imports from `four` imports from `one` for a depth of four. Adding the option to `tslint.json` means these shouldn't cause a lint complaint, which is why I didn't modify the list of expected complaints.